ui/src/alerting/components/builder/ThresholdConditions.tsx

```diff
@@ -0,0 +1,61 @@
+// Libraries
+import React, {FC} from 'react'
+import {connect} from 'react-redux'
+
+// Components
+import {
+  FlexBox,
+  FlexDirection,
+  AlignItems,
+  ComponentSize,
+} from '@influxdata/clockface'
+import ThresholdCondition from 'src/alerting/components/builder/ThresholdCondition'
+
+// Actions
+import {getActiveTimeMachine} from 'src/timeMachine/selectors'
+
+// Types
+import {Threshold, AppState} from 'src/types'
+
+interface ThresholdsObject {
+  [k: string]: Threshold
+}
+
+interface StateProps {
+  thresholds: ThresholdsObject
+}
+
+const ThresholdConditions: FC<StateProps> = ({thresholds}) => {
+  return (
+    <FlexBox
+      direction={FlexDirection.Column}
+      alignItems={AlignItems.Stretch}
+      margin={ComponentSize.Medium}
+    >
```

Comment:
`const LEVELS = ["OK"...]`
```suggestion
    >
    {LEVELS.map((level) => <ThresholdCondition level={level} thresholds={thresholds[level]} />}
```